subsys/bluetooth/host/gatt.c

```diff
@@ -2542,20 +2530,40 @@ int bt_gatt_notify_cb(struct bt_conn *conn,
 int bt_gatt_notify_multiple(struct bt_conn *conn, uint16_t num_params,
 			    struct bt_gatt_notify_params *params)
 {
-	int i, ret;
-
 	__ASSERT(params, "invalid parameters\n");
-	__ASSERT(num_params, "invalid parameters\n");
-	__ASSERT(params->attr, "invalid parameters\n");
+	__ASSERT(num_params >= 2, "invalid parameters\n");
```

Comment:
The spec allows sending a single notification in a notify_multiple. Should we not allow it?